docs/pipelines.md

```diff
@@ -375,7 +375,7 @@ tasks:
             resource: source-repo
 ```
 
-In this example, `my-condition` refers to a [Condition](#conditions) custom resource. The `build-push` 
+In this example, `my-condition` refers to a [Condition](./conditions.md) custom resource. The `build-push` 
```

Comment:
nit: It can just be `conditions.md`

```suggestion
In this example, `my-condition` refers to a [Condition](conditions.md) custom resource. The `build-push` 
```